### ISSUE TITLE
How to train without pretrained checkpoint

### ISSUE THREAD

**Issue by ydhongHIT** [HiKapok/SSD.TensorFlow#105]
I see that author's answer is 'scaffold = None' and I try it. But the loss at the beginning is only from 20 to 30. I think the loss is too low. Really train without pretrained checkpoints?
